arbeitszeit_flask/translations/de/LC_MESSAGES/messages.po

```diff
@@ -2124,6 +2198,10 @@ msgstr "Unerwarteter Fehler: Nicht möglich, die Planeinreichung zu wiederrufen.
 msgid "Filing of plan has been successfully revoked."
 msgstr "Planeinreichung wurde erfolgreich widerrufen."
 
+#: arbeitszeit_web/www/presenters/select_productive_consumption_presenter.py:44
+msgid "The selected plan does not exist or is not active anymore."
+msgstr "Der ausgewählte Plan existiert nicht oder ist nicht mehr aktiv."
```

Comment:
how can this happen? A plan can be selected, but does not exist?

Reply:
See this discussion (https://github.com/ida-arbeitszeit/arbeitszeitapp/issues/1053) and the screenshot below. We agreed on having a two-stage productive consumption process: First the user "selects" a plan and if this plan exists more details are shown. So it may happen that a user selects a plan that does not exist.

![alt text](<Bildschirmfoto vom 2024-11-02 15-50-41.png>)

Reply:
I see, thanks! so the plan id is not really selected, but instead it is a free-text field. In this case, sth like "a plan with this plan id does not exist" would be clearer. but such a minor issue can be taken up later imo.

Reply:
yes, you are right.